versions/3.1.0.md

```diff
@@ -161,14 +163,14 @@ The formats defined by the OAS are:
 `integer` | `int64` | signed 64 bits (a.k.a long)
 `number` | `float` | |
 `number` | `double` | |
-`string` | | |
-`string` | `byte` | base64 encoded characters
-`string` | `binary` | any sequence of octets
-`boolean` | | |
-`string` | `date` | As defined by `full-date` - [RFC3339](https://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14)
-`string` | `date-time` | As defined by `date-time` - [RFC3339](https://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14)
 `string` | `password` | A hint to UIs to obscure input.
+`string` | `byte` | base64 encoded characters **Deprecated**
+`string` | `binary` | any sequence of octets **Deprecated**
```

Comment:
Should these simply be removed?  What would the impact be on other content-types?  If someone is using JSON Schema to infer schema on other non-JSON formats, what does format even mean?

Reply:
Personally, I think we can remove these.